Pipfile

```diff
@@ -5,12 +5,11 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
 
 [packages]
```

Comment:
You have it in [dev-packages]
I think django is missing here, did you check if your server is running after vagrant up?